presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveDistributedJoinQueriesWithoutDynamicFiltering.java

```diff
@@ -31,12 +32,14 @@
 import io.prestosql.testing.ResultWithQueryId;
 import org.testng.annotations.Test;
 
-import static io.prestosql.SystemSessionProperties.ENABLE_DYNAMIC_FILTERING;
 import static io.prestosql.SystemSessionProperties.JOIN_DISTRIBUTION_TYPE;
 import static io.prestosql.tpch.TpchTable.getTables;
 import static org.testng.Assert.assertEquals;
 
-public class TestHiveDistributedJoinQueriesWithDynamicFiltering
+/**
+ * @see TestHiveDistributedJoinQueries for tests with dynamic filtering enabled
+ */
+public class TestHiveDistributedJoinQueriesWithoutDynamicFiltering
```

Comment:
I think we can remove it. There is no big benefit of this tests.

Reply:
I cannot say I agree because of https://github.com/prestosql/presto/issues/3149#issuecomment-601214833

Reply:
again I think it's unneeded. For example, we test joins with spill only in the engine: `io.prestosql.tests.TestSpilledJoinQueries`